### ISSUE TITLE
servicesDirectory option does not function correctly

### ISSUE THREAD

**Issue by lageorgem** [mj1618/serverless-offline-sns#144]
The issue: 
When including the servicesDirectory option, the `loadServerlessConfig` file is called with the argument `shell.pwd()` which appears to return a non-primitive `String`. This non-primitive is then passed to `fs.statSync` which fails.

```
debug loadServerlessConfig [String: '/path/to/other/project'] {
  stdout: '/path/to/other/project',
  stderr: null,
  code: 0,
  cat: [Function: bound ],
  exec: [Function: bound ],
  grep: [Function: bound ],
  head: [Function: bound ],
  sed: [Function: bound ],
  sort: [Function: bound ],
  tail: [Function: bound ],
  to: [Function: bound ],
  toEnd: [Function: bound ],
  uniq: [Function: bound ]
}
 
 Type Error ----------------------------------------------
 
  TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received an instance of String
      at Object.statSync (fs.js:1120:10)
      at Object.statSync (/path/to/project/node_modules/graceful-fs/polyfills.js:312:16)
      at Object.<anonymous> (/path/to/project/node_modules/serverless-offline-sns/dist/sls-config-parser.js:125:31)
      at step (/path/to/project/node_modules/serverless-offline-sns/dist/sls-config-parser.js:59:23)
      at Object.next (/path/to/project/node_modules/serverless-offline-sns/dist/sls-config-parser.js:40:53)
      at /path/to/project/node_modules/serverless-offline-sns/dist/sls-config-parser.js:34:71
      at new Promise (<anonymous>)
      at __awaiter (/path/to/project/node_modules/serverless-offline-sns/dist/sls-config-parser.js:30:12)
      at Object.loadServerlessConfig (/path/to/project/node_modules/serverless-offline-sns/dist/sls-config-parser.js:119:12)
      at /path/to/project/node_modules/serverless-offline-sns/dist/index.js:251:82
      at step (/path/to/project/node_modules/serverless-offline-sns/dist/index.js:33:23)
      at Object.next (/path/to/project/node_modules/serverless-offline-sns/dist/index.js:14:53)
      at step (/path/to/project/node_modules/serverless-offline-sns/dist/index.js:18:139)
      at Object.next (/path/to/project/node_modules/serverless-offline-sns/dist/index.js:14:53)
      at fulfilled (/path/to/project/node_modules/serverless-offline-sns/dist/index.js:5:58)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              14.17.4
     Framework Version:         2.65.0 (local)
     Plugin Version:            5.5.1
     SDK Version:               4.3.0
     Components Version:        3.17.2

```

**Comment by ThMrtns** [comment]
I fall into the same. Anybody knows about a workaround?

**Comment by 1frag** [comment]
Fixed errors in #148.

**Comment by dsarlo** [comment]
This has been resolved
